RepClient/src/app/services/api-endpoint-creator.service.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { TestBed } from '@angular/core/testing';
+
+import { ApiEndpointCreatorService } from './api-endpoint-creator.service';
+
+describe('ApiEndpointCreatorService', () => {
+  beforeEach(() => TestBed.configureTestingModule({}));
+
+  it('should be created', () => {
+    const service: ApiEndpointCreatorService = TestBed.get(ApiEndpointCreatorService);
+    expect(service).toBeTruthy();
+  });
```

Comment:
Additional tests for registerEndPoint and loginEndPoint would be beneficial here.